course_content/cartopy_course/cartopy_intro.ipynb

```diff
@@ -119,29 +143,157 @@
    "source": [
     "ax = plt.axes(projection=ccrs.PlateCarree())\n",
     "ax.coastlines()\n",
+    "x0, y0 = -50, -30\n",
+    "x1, y1 = 10, 55\n",
+    "plt.plot([x0, x1], [y0, y1], linewidth=4)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's try plotting the same line on an `EquidistantConic` projection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proj = ccrs.EquidistantConic()\n",
+    "\n",
+    "ax = plt.axes(projection=proj)\n",
+    "ax.coastlines()\n",
+    "plt.plot([x0, x1], [y0, y1], linewidth=4)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above plot is ***incorrect***.  \n",
+    "We have set up the axes to be in the Orthographic projection, but we have not told Cartopy that the coordinates of the line are \"in PlateCarree\".  \n",
+    "To do this, we use the `transform` keyword in the plt.plot function :\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = plt.axes(projection=proj)\n",
+    "ax.coastlines()\n",
+    "plt.plot([x0, x1], [y0, y1], linewidth=4, transform=ccrs.PlateCarree())\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the plotted line is bent :  It is a straight line ***in the coordinate system it is defined in***, so that makes it a curved line on this map.\n",
+    "\n",
+    "Also note that, unless we specify a map extent, the map zooms to contain just the plotted data.  \n",
+    "A very simple alternative to that is to plot the 'full map', by calling the **``set_global``** method on the Axes, as in this case :"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = plt.axes(projection=proj)\n",
+    "ax.coastlines()\n",
     "ax.set_global()\n",
-    "plt.plot([-100, 50], [25, 25], linewidth=4, transform=ccrs.Geodetic())\n",
+    "plt.plot([x0, x1], [y0, y1], linewidth=4, transform=ccrs.PlateCarree())\n",
     "plt.show()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Notice that unless we specify a map extent (we did so via the **``set_global``** method in this case) the map will zoom into the range of the plotted data."
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color='brown'>Suggested Activity :</font></b> Try modifying the above code to plot a different projection.\n",
+    "\n",
+    "Try re-plotting the \"failed plot\" above, where we did not specify the transform.<br>\n",
+    "But now, add a call to the `set_global` method, to show the full map extent.\n",
+    "\n",
+    "What does this tell you about what was actually being plotted in that case ?\n",
```

Comment:
This now reads as:
```
Try modifying the above code to plot a different projection. Try re-plotting the "failed plot" above, where we did not specify the transform.

But now, add a call to the `set_global` method, to show the full map extent. What does this tell you about what was actually being plotted in that case ? 
```

It looks like the first sentence was a mistake in copying from the first "suggested activity"

Reply:
I find the sentence starting `But now a bit strange`
It reads as:
>Try re-plotting the "failed plot" above, where we did not specify the transform.
>But now, add a call to the `set_global` method, to show the full map extent. What does this tell you about what was actually being plotted in that case ? 


I find it strange you split the two sentences up. How about
>Try re-plotting the "failed plot" above, where we did not specify the transform. but this time add a call to the `set_global` method, to show the full map extent. What does this tell you about what was actually being plotted in that case ?

Reply:
I'm trying to get a short initial 'title'-like sentence, then more if needed.
I will try to improve ...
(see new version upcoming)

Reply:
Definitely prefer your new version :+1: